front/src/styles/scss/applications/stdcmV2/_card.scss

```diff
@@ -0,0 +1,44 @@
+.stdcm-v2-card {
+    border-radius: 8px;
+    box-shadow: 0px 1px 0px rgba(255, 255, 255, 1) inset, 0px 4px 7px -3px rgba(24, 68, 239, 0.19), 0px 1px 3px rgba(49, 46, 43, 0.2);
+    background-color: rgba(255, 255, 255, 1);
+    height: fit-content;
+    position: relative;
+
+    &.has-tip::after {
+        content: "";
+        position: absolute;
+        top: 100%;
+        left: 20px;
+        border-top: 20px solid transparent;
+        border-top-color: rgba(255, 255, 255, 1); 
+        border-left: 20px solid transparent;
+        border-right: 20px solid transparent; 
+
+    }
+
+    .stdcm-v2-card__header {
+        border-radius: 8px 8px 0px 0px;
+        box-shadow: 0px 1px 0px rgba(255, 255, 255, 0.75) inset;
+        background-color: rgba(235, 235, 234, 1);
+        color: rgba(0, 0, 0, 1);
+        // font-family: "IBMPlexSans-SemiBold";
```

Comment:
Why is this commented ? It looks like the font used doesn't match exactly the one from the model.

If the font isn't ready yet, maybe we should leave a comment or TODO here.